docopts.go

```diff
@@ -17,7 +17,6 @@ import (
 )
 
 // vars defined at compile time
-// https://github.com/ahmetb/govvv
```

Comment:
may be a link to goreleaser?

Reply:
So, previously these were variables defined by `govvv`, but `goreleaser` doesn't define these.

I manually passed these in as `ldflags` in the `.goreleaser.yml`:
https://github.com/docopt/docopts/pull/65/files#diff-7326b55c062b0f46fe9e39aace0a25f4515cf206040fb91a6fd2cae839f5e826R21
There I have a bunch of comments.

So these are just regular `ldflags` defined at compile-time; it's not specific to `goreleaser` and they could be named anything (vs. those were specific names that `govvv` used before).